app/src/main/java/com/cyberbot/checkers/game/Grid.java

```diff
@@ -1,5 +1,7 @@
 package com.cyberbot.checkers.game;
 
+import android.net.wifi.p2p.WifiP2pManager;
```

Comment:
No Android related code in game package. Looks like an unused import.

Reply:
removed